samples/multiplatform/platform-common/src/test/kotlin/Tests.kt

```diff
@@ -0,0 +1,26 @@
+import sample.Constant
```

Comment:
the package is wrong, no?

Reply:
Well `Constant`  class defined in sample package, what is wrong?